libs/agno/agno/vectordb/milvus/milvus.py

```diff
@@ -207,7 +207,7 @@ def _prepare_hybrid_index_params(self) -> Any:
             index_name="sparse_index",
             index_type="SPARSE_INVERTED_INDEX",
             metric_type="IP",
-            params={"drop_ratio_build": 0.2},
+            params={"drop_ratio_search": 0.2},
```

Comment:
The purpose of this function is building index params, and `drop_ratio_build` is  used during the index building process for sparse vectors

I dont think this one has to be changed